pymeasure/adapters/prologix.py

```diff
@@ -37,8 +37,10 @@ class PrologixAdapter(VISAAdapter):
     Serial connection sharing is achieved by using the :meth:`.gpib`
     method to spawn new PrologixAdapters for different GPIB addresses.
 
-    :param port: The Serial port name or a connection object
-    :param address: Integer GPIB address of the desired instrument
+    :param resource_name: A
```

Comment:
I would consider to leave the name as `port` or similar to remove ambiguity with resource name term used in `Instrument` class.

Reply:
We don't use "resource_name" in the instrument anymore.
On the other hand, the visa Adapter (superclass) does use it and I tried to keep the name scheme of the superclass.

Besides, the connection could be ethernet etc, where resource name is better than port.

Reply:
This an excerpt of Instrument documentation and we use the term resource name (correctly in my opinion)

> When adapter is a string, this is taken as an appropriate resource name. Depending on your installed VISA library, this can be something simple like COM1 or ASRL2, or a more complicated VISA resource name defining the target of your connection.
>
> When adapter is an integer, a GPIB resource name is created based on that. In either case a VISAAdapter is constructed based on that resource name. Keyword arguments can be used to further configure the connection.

> On the other hand, the visa Adapter (superclass) does use it and I tried to keep the name scheme of the superclass.

Indeed that's was my point, the usage intended in `VISAAdapter` is totally different than the one intended in `PrologixAdapter`, but your latest documentation update clarify this.

Thanks.